apps/06SpaceTravelToMars.js

```diff
@@ -9,19 +9,66 @@ var jobTypes = {
   programmer: 'Any Ship!'
 };
 
-//**************************************************************************************************************************
 // Your code here
 
+function CrewMember(name, job, specialSkill){
+  this.name = name;
+  this.job = job;
+  this.specialSkill = specialSkill;
+  this.ship = null;
```

Comment:
Hey Rey! Try to push this one again...seems not to be passing the tests but I am sure I have seen it pass before!